org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -57,6 +57,14 @@
 	 * Specifies the folder path to the JDK .
 	 */
 	public static final String JAVA_HOME = "java.home";
+	/**
+	 * Insert spaces when pressing Tab
+	 */
+	public static final String JAVA_CONFIGURATION_INSERTSPACES = "java.editor.insertSpaces";
+	/**
+	 * Tab Size
+	 */
+	public static final String JAVA_CONFIGURATION_TABSIZE = "java.editor.tabSize";
```

Comment:
`java.format. tabSize` would be more consistent IMO

Reply:
Fixed.